src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -343,6 +343,15 @@ private void ChooseActions(in Entity entity, ref MicrobeAI ai, ref CompoundAbsor
             }
         }
 
+        // If motionless inside a hydrogen sulfide cloud, move away
+        if (!organelles.HydrogenSulfideProtection
+            && compounds.GetCompoundAmount(Compound.Hydrogensulfide) > MathUtils.EPSILON
+            && control.MovementDirection != Vector3.Zero)
```

Comment:
Isn't this last condition the wrong way around? Doesn't this now check that if the cell *is* moving?